pkg/ebpf/events_pipeline.go

```diff
@@ -46,6 +46,12 @@ func (t *Tracee) handleEvents(ctx context.Context) {
 	eventsChan, errc = t.processEvents(ctx, eventsChan)
 	errcList = append(errcList, errc)
 
+	eventsChan, errc = t.processTree.StartProcessingPipeline(ctx, eventsChan)
```

Comment:
Add this stages to the events processing stage.
This will require to create a solution to get all events in the processing - to be able to fill default process data in the tree.